openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -719,6 +719,15 @@
       // genereal feedback question
       .assessment__rubric__question--feedback {
 
+        .wrapper--input {
+          margin-top: $baseline-v;
+        }
+
+        .question__title__copy {
+          @include margin-left(0);
```

Comment:
This is what was causing the odd indentation for self, it was already present for peer: https://github.com/edx/edx-ora2/blob/eff9961015d1164e2b119c7eb9a91e6580740c8d/openassessment/xblock/static/sass/oa/views/_oa-base.scss#L658